src/ensembl/src/content/app/browser/BrowserCogList.tsx

```diff
@@ -36,40 +32,36 @@ type DispatchProps = {
   updateSelectedCog: (index: string) => void;
 };
 
-type OwnProps = {
-  browserRef: RefObject<HTMLDivElement>;
-};
+type OwnProps = {};
 
 type BrowserCogListProps = StateProps & DispatchProps & OwnProps;
 
-type BpaneScrollEvent = Event & {
-  detail: {
-    delta_y?: number;
-    track_y?: CogList;
-  };
+type BpaneScrollPayload = {
+  delta_y?: number;
+  track_y?: CogList;
 };
 
 const BrowserCogList: FunctionComponent<BrowserCogListProps> = (
   props: BrowserCogListProps
 ) => {
   const { browserCogTrackList } = props;
-  const listenBpaneScroll = useCallback((event: Event) => {
-    const bpaneScrollEvent = event as BpaneScrollEvent;
-    if (
-      bpaneScrollEvent.detail.delta_y ||
-      bpaneScrollEvent.detail.delta_y === 0
-    ) {
-      props.updateCogList(bpaneScrollEvent.detail.delta_y);
+  const listenBpaneScroll = (payload: BpaneScrollPayload) => {
+    const { delta_y, track_y } = payload;
+    if (typeof delta_y === 'number') {
```

Comment:
Do we need to check if it is a number here as it will always be a number if it is defined?

Reply:
It can be a number of undefined. The check above did 

```
if (
      bpaneScrollEvent.detail.delta_y ||
      bpaneScrollEvent.detail.delta_y === 0
    )
```
because if delta_y is 0 it will not pass the test of `if (delta_y)` (because 0 is falsy). But we want 0 to make it to the body of the condition block.

So our options are:
- either check `if (delta_y || delta_y ===0)`, which is confusing
- or check `if (typeof delta_y === 'number')`, which I did
- or check `if (delta_y !== undefined)`

If you prefer the third variant of check to the second one, then I am happy to change it.

Reply:
Can `delta_y` have negative values? 
If yes, we could check `if (delta_y > -1)`

If not, I think the third variant will be better than the second one.

Reply:
Replaced type check with a check for `undefined`.